tags/lib/mixinextras.ytag

```diff
@@ -0,0 +1,9 @@
+type: text
+
+---
+
+Mixin Extras is a small companion library for the Mixin system, offering better ways to use Mixins.
```

Comment:
Not necessarily better, just more options. It won't help you if all you need is a simple `@Inject`.

Reply:
Replaced "better" with "alternative"